core/domain/wipeout_service_test.py

```diff
@@ -454,15 +454,17 @@ def test_pre_delete_user_exploration_ownership_is_released_with_voice_art(
             'exp_id',
             self.user_2_id,
             feconf.ROLE_VOICE_ARTIST)
+        rights_manager.unpublish_exploration(
+            user_services.get_system_user(), 'exp_id')
 
         exp_summary_model = exp_models.ExpSummaryModel.get_by_id('exp_id')
-        self.assertFalse(exp_summary_model.community_owned)
+        self.assertEqual(exp_summary_model.voice_artist_ids, [self.user_2_id])
 
-        wipeout_service.pre_delete_user(self.user_1_id)
+        wipeout_service.pre_delete_user(self.user_2_id)
         self.process_and_flush_pending_tasks()
 
         exp_summary_model = exp_models.ExpSummaryModel.get_by_id('exp_id')
-        self.assertTrue(exp_summary_model.community_owned)
```

Comment:
Can you explain the change in this part? (Why was it this way before? I'm not quite sure I understand the reasoning.)

Reply:
The test was wrong previously. By accident, it also passed before.

Reply:
Can you explain that a bit more please -- how did it pass "by accident"?

Reply:
Ah, sorry, this was a mistake in the end. Changed it back.